grappelli/dashboard/utils.py

```diff
@@ -5,7 +5,7 @@
 """
 
 # PYTHON IMPORTS
-from __future__ import unicode_literals
```

Comment:
Sorry was not intentional. `2to3` tool removed these imports. I will refactor this shortly.